app/connector/BackendConnector.scala

```diff
@@ -36,9 +38,18 @@ class BackendConnector @Inject()(
   private val urlMetadata = s"${bars.baseUrl}/metadata/"
   private val urlAssess = s"${bars.baseUrl}/assess"
 
-  def validate(account: AccountDetails)(implicit hc: HeaderCarrier): Future[ValidationResult] = {
+  implicit val sensibleReads: HttpReads[HttpResponse] = new HttpReads[HttpResponse] {
+    def read(method: String, url: String, response: HttpResponse) = response
+  }
+
+  def validate(account: AccountDetails)(implicit hc: HeaderCarrier): Future[Either[ValidationErrorResult, ValidationResult]] = {
 
-    http.POST(urlValidate, account).map(response => response.json.validate[ValidationResult].get)
+    http.POST[AccountDetails, HttpResponse](urlValidate, account).map{
+      case response if response.status == 200 =>
+        Right(response.json.validate[ValidationResult].get)
+      case response if response.status == 400 =>
+        Left(response.json.validate[ValidationErrorResult].get)
+    }
```

Comment:
Is it possible for 404, or 500 to come in here?  Should we have a default to deal with unexpected error codes?